### ISSUE TITLE
Issue 8 — Entrenamiento con validación

### ISSUE THREAD

**Issue by Alaedddine718** [Alaedddine718/Deep-Learning_NV#9]
Issue 8 — Entrenamiento con validación

Labels: train, priority:high
Descripción: Entrenar ≥10 épocas con validation_split=0.1.
Checklist

 history = model.fit(..., epochs=10, batch_size=64, validation_split=0.1).

 Guardar history.history.
DoD

 Log de entrenamiento completo y guardado.
